### ISSUE TITLE
Add support for implicit-casts: false & implicit-dynamic: false

### ISSUE THREAD

**Issue by esDotDev** [bnxm/dart-data-class-generator#22]
There is another issue around lint rules, but I think that these two in particular are almost a necessity: 
    implicit-casts: false
    implicit-dynamic: false

Without these turned off, your dart code can get very sloppy. It should be easy to directly cast things to their value types?

**Comment by bnxm** [comment]
Yes I will probably do this, especially because I think those rules will become the default with NNBD. 

> It should be easy to directly cast things to their value types?

Yes it's easy, however, it does add more boilerplate which is why I left it out so far.

**Comment by sgehrman** [comment]
still waiting for this fix.

**Comment by devmuaz** [comment]
@bnxm Any updates for this issue?
